### DISCUSSION TITLE
上位机软件，在win10上打开很卡顿

### DISCUSSION THREAD

**Discussion by dingdingrao** [peng-zhihui/ElectronBot#149] [Q&A]
上位机软件，在win10上打开很卡顿，动作调整拖动都无法顺利完成，这是什么原因呢？和集显有关系吗？

**Comment by gungun678** [comment]
上位机一般只吃内存和读写，不吃显卡
电脑配置实在不行的话，时间要求不高的话，可以看看设置有没有扫描时间设置，调长点

**Reply by dingdingrao** [reply]
我感觉不是电脑内存和硬盘问题呢，我是10G内存，SSD硬盘，目前应到的大软件都不卡顿。

**Comment by coolflyreg** [comment]
毕竟是3D游戏引擎做的。显卡不好确实不行

**Reply by dingdingrao** [reply]
我感觉也是显卡问题呢
